src/features/AddNoteForm/AddNoteForm.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { expect } from "@storybook/jest"
+import { ComponentMeta, ComponentStory } from "@storybook/react"
+import { within } from "@storybook/testing-library"
+import AddNoteForm from "./AddNoteForm"
+
+export default {
+  title: "Features/CourtCase/Notes/Add",
+  component: AddNoteForm
+} as ComponentMeta<typeof AddNoteForm>
+
+export const WhenCaseNotLockedByAnotherUser: ComponentStory<typeof AddNoteForm> = () => (
```

Comment:
Small thing - but you could write some extra tests here checking that you can actually write a note in, hit the `Add` button, and that it then is saved (rendered on screen correctly)?

Reply:
Just feels a bit odd to me that a note-taking component doesn't have stories around it taking notes 📓